public/v4/apps/yourls.yml

```diff
@@ -0,0 +1,82 @@
+        restart: always
+        environment:
+            MYSQL_ROOT_PASSWORD: $$cap_db_pass
+            MYSQL_DATABASE: $$cap_YOURLS_DB_NAME
+            MYSQL_USER: $$cap_db_user
+            MYSQL_PASSWORD: $$cap_db_pass
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+    $$cap_appname:
+        depends_on:
+            - $$cap_appname-db
+        image: yourls:$$cap_yourls_version
+        volumes:
+            - $$cap_appname-logs:/usr/src/yourls/logs
+            - $$cap_appname-plugins:/usr/src/yourls/plugins
+        restart: always
+        environment:
+            YOURLS_SITE: $$cap_YOURLS_SITE
+            YOURLS_USER: $$cap_YOURLS_USER
+            YOURLS_PASS: $$cap_YOURLS_PASS
+            YOURLS_DB_HOST: srv-captain--$$cap_appname-db:3306
+            YOURLS_DB_NAME: $$cap_YOURLS_DB_NAME
+            YOURLS_DB_USER: $$cap_db_user
+            YOURLS_DB_PASS: $$cap_db_pass
+caproverOneClickApp:
+    variables:
+        - id: $$cap_YOURLS_SITE
+          label: YOURLS instance URL
+        - id: $$cap_YOURLS_USER
+          label: YOURLS instance username
+        - id: $$cap_YOURLS_PASS
+          label: YOURLS instance password
+          description: ''
+        - id: $$cap_YOURLS_PRIVATE
+          label: Loginprotected admin area
+          defaultValue: true
+        - id: $$cap_YOURLS_UNIQUE_URLS
+          label: Allow multiple short URLs for a same long URL
+          defaultValue: true
+        - id: $$cap_YOURLS_COOKIEKEY
+          label: A random secret hash used to encrypt cookies
+          defaultValue: $$cap_gen_random_hex(28)
+        - id: $$cap_YOURLS_DB_NAME
+          label: Database name
+          defaultValue: yourls
+        - id: $$cap_db_user
+          label: Database user
+          defaultValue: yourls
+          validRegex: /^([a-zA-Z0-9])+$/
+        - id: $$cap_db_pass
+          label: Database password
+          defaultValue: $$cap_gen_random_hex(16)
+          description: ''
+          validRegex: /^(\w|[^\s"'\\])+$/
+        - id: $$cap_database_version
+          label: Database Version
+          defaultValue: '5.7'
+          description: Check out the Docker pages for the valid tags https://hub.docker.com/r/library/mysql/tags/
+          validRegex: /^([^\s^\/])+$/
+        - id: $$cap_yourls_version
+          label: yourls Version
+          defaultValue: '1.9.2'
+          description: Check out their Docker page for the valid tags https://hub.docker.com/r/library/yourls/tags/
+          validRegex: /^([^\s^\/])+$/
+    instructions:
+        start: >-
+            YOURLS is a set of PHP scripts that will allow you to run Your Own URL Shortener.
+        end: >
+            yourls is ready for final steps at $$cap_YOURLS_SITE/admin .
+            In case you have set another Instance URL than http://$$cap_appname.$$cap_root_domain: Don't forget to connect! (and enable HTTPS)
+
+            Have fun!
+    displayName: yourls
+    isOfficial: false
```

Comment:
I think you're using the official image.

Reply:
True! Changed...